README.md

```diff
@@ -1,90 +1,141 @@
-# Django proof-of-concept for CodeBuddies V3
+# [CodeBuddies V3 Back-end]
+## Django proof-of-concept
 
 ![Test](https://github.com/codebuddies/django-concept/workflows/Test/badge.svg)
 [![codecov](https://codecov.io/gh/codebuddies/backend/branch/master/graph/badge.svg)](https://codecov.io/gh/codebuddies/backend)
 
+**Note:** This project is currently _in development_
 
+## Features
 
+- **Auto-reload** - modify the application code in your editor of choice. As you save changes, the application should reload automatically. There should be no need to restart containers to see code changes.
+<!-- TODO -->
 
-Background: https://github.com/codebuddies/codebuddies/issues/1136
+## Reference
 
-The API spec all the proof-of-concepts: https://app.swaggerhub.com/apis-docs/billglover/CodeBuddies/0.0.1
+- [Background]
+- [The API spec all the proof-of-concept]
+- [Crowdsourced brainstorm of problems we want to solve]
+- Will interact as the API supporting the [Codebuddies V3 Front-end]
 
-Crowdsourced brainstorm of problems we want to solve: https://pad.riseup.net/p/BecKdThFsevRmmG_tqFa-keep
+## Getting started
 
-## Setup
-
-Although it is possible to run this locally, we recommend you run CodeBuddies locally using Docker. We assume you have Docker installed, but if not head on over to the Docker [Getting Started](https://www.docker.com/products/docker-desktop) guide and install Docker for your operating system.
+We recommend you run CodeBuddies locally using Docker. We assume you have Docker installed, but if not head on over to the Docker [Getting Started] guide and install Docker for your operating system.
 
 These instructions have been used on the following operating systems.
 
-* Linux
-* Mac OS
-* Windows 10 Pro
-
-Please note that Windows 10 Home is not supported by Docker Desktop at this time.
+- Linux
+- Mac OS
+- Windows 10 Pro - Please note that Windows 10 Home is not supported by Docker Desktop at this time.
```

Comment:
Because I've been AFK from this project for a bit--we can't use Windows 10 Home because the Docker for Windows doesn't support it, correct?

Reply:
I've been AFK for a while too, but recently learned the good news that Docker for Windows 10 Home is now supported! :) https://docs.docker.com/docker-for-windows/install-windows-home/

Reply:
We'll probably have to do one more README update PR to clean this up.

Reply:
just some personal experiences from work about docker using on Windows: docker commands on Windows Subsystem Linux (WSL)  and regular Windows (10 Pro) could be different as WSL is completely Linux environment. But I agree WSL is definitely a work around for Windows 10 Home user to use docker commands.

on the other hand its a bit tricky to do certain things i.e. install packages from links (at least for me - i blame ubuntu20.04 being too new) as WSL still mainly relies on Windows system at the very back. I'm still trying to figure out about it for work purpose. will test out docker commands on readme and give feedback if there's any issues.

Reply:
Thanks @nuageklow! Also thank you for your comments elsewhere on this PR. Are you able to use the bash terminal for commands, or is it still relying on powershell?

Reply:
@bengineerdavis WSL settings like setting it up relies on powershell (like an app) but it uses bash terminal